app/views/layouts/account.html.erb

```diff
@@ -0,0 +1,75 @@
+<% content_for :body do %>
+  <%= yield :back_link %>
+  <main role="main" id="content">
+    <%= yield %>
+  </main>
+<% end %>
+
+<%= content_for :head do %>
+  <%= csp_meta_tag %>
+  <meta name="robots" content="noindex, nofollow">
+
+  <% add_gem_component_stylesheet("cookie-banner") %>
+  <% add_gem_component_stylesheet("cross-service-header") %>
+  <% add_gem_component_stylesheet("emergency-banner") %>
+  <% add_gem_component_stylesheet("layout-footer") %>
+  <% add_gem_component_stylesheet("layout-for-public") %>
+  <% add_gem_component_stylesheet("phase-banner") %>
+  <% add_gem_component_stylesheet("skip-link") %>
+  <%= render_component_stylesheets %>
+<% end %>
+
+<%= render "govuk_publishing_components/components/layout_for_public", {
+  title: yield(:title),
+  blue_bar: false,
+  emergency_banner: render("govuk_web_banners/emergency_banner"),
+  omit_feedback_form: true,
+  omit_footer_border: true,
+  omit_footer_navigation: true,
+  product_name: "GOV.UK email subscriptions",
+  show_account_layout: true,
+  show_cross_service_header: true,
+  show_explore_header: false,
```

Comment:
non-blocking: we should be able to delete this line now, the `show_explore_header` option was removed from layout_for_public in [v54.0.0](https://github.com/alphagov/govuk_publishing_components/blob/main/CHANGELOG.md#5400) of govuk_publishing_components